### PR TITLE
added apsw 3.16.2 for use with sqlite 3.16.2

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

**Comment by stuertz** [comment]
@conda-forge/staged-recipes Ready for review/merge :-) !

**Comment by stuertz** [comment]
@ocefpaf This is ready for merge, please have a look.

File: recipes/apsw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/rogerbinns/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py fetch --sqlite --version {{ sqlite_version }}
```
Comment (ocefpaf) [inline recipes/apsw/meta.yaml]: Did you try the `sqlite` from `conda-forge`?

Comment (stuertz) [inline recipes/apsw/meta.yaml]: No, I didn't. 
Thought it is a good idea, to use a pinned version of sqlite, that is always the same version as apsw. This way apsw is independent of the conda-forge sqlite package.

I could give this a try, if requested ;-)

File: recipes/apsw/meta.yaml [hunk 2]
```diff
@@ -15,14 +15,14 @@ source:
...
   script:
-    - python setup.py fetch --sqlite --version {{ sqlite_version }}  
     - python setup.py install --single-version-externally-managed --record record.txt
     - python setup.py test
   
 requirements:
   build:
     - python
     - setuptools
+    - sqlite {{ sqlite_version }}
```
Comment (ocefpaf) [inline recipes/apsw/meta.yaml]: Please don't pin with a jinja variable b/c that will probably break the conda-forge's pinning bot.

**Comment by stuertz** [comment]
Hi @ocefpaf,
sorry for the delay, I was on vacation, and had some other projects :-)
I now have finally changed to use sqlite from conda instead of the amalgamation code. 

There is still an issue with the test / ci builder on macos, which loads the wrong sqlite so from system. So I decided to skipped macos for now. I'll have a look later when I have my fingers back on a mac console, whether thats a real issue, or just a problem with the ci test.

So for now please have a kindly look. ;-)

**Comment by ocefpaf** [comment]
@stuertz now I am the one traveling. Maybe someone from @conda-forge/staged-recipes can help you here.

File: recipes/apsw/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  # Currently the import test fails on osx, because wrong sqlite3 so lib is
+  # found. This needs more investigation.
+  skip: True  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sqlite 3.16.2
```
Comment (isuruf) [inline recipes/apsw/meta.yaml]: Add toolchain here

File: recipes/apsw/build.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include -L$PREFIX/lib"
```
Comment (isuruf) [inline recipes/apsw/build.sh]: Change this to
```
export CFLAGS="-I$PREFIX/include ${CFLAGS}"
export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib ${LDFLAGS}"
```
This should probably fix the OSX build.

**Pull request merged**